### PR TITLE
Introduce optional <optgroup> label to DropdownItem

### PR DESCRIPTION
**Pull request by joaocanaverde**
## What does this do?

Introduces optional [<optgroup>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup) labels to dropdown items.

### CODE REVIEW

File: src/Dropdown/Dropdown.stories.js [hunk 1]
```diff
@@ -49,6 +87,14 @@ Default.args = {
   placeholder: 'Select Day',
 }
 
+export const DefaultWithGroups = Template.bind({})
+
+DefaultWithGroups.args = {
+  id: 'days',
+  list: daysWithGroups,
```
Comment (fleonard) [inline src/Dropdown/Dropdown.stories.js:94]: Maybe better to keep the default one without groups?

**Review by fleonard — CHANGES_REQUESTED** [review]
Looks good, can you please edit the CHANGELOG file so it will populate the release once you bump version and push tags?

File: src/Dropdown/Dropdown.stories.js (hunk 1 above)
Comment (joaocanaverde) [reply src/Dropdown/Dropdown.stories.js:94]: @fleonard We've got both still:

```javascript
export const Default = Template.bind({})

Default.args = {
  id: 'days',
  list: days,
  placeholder: 'Select Day',
}

export const DefaultWithGroups = Template.bind({})

DefaultWithGroups.args = {
  id: 'days',
  list: daysWithGroups,
  placeholder: 'Select Day',
}
```

Comment (fleonard) [reply src/Dropdown/Dropdown.stories.js:94]: missed it

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+## [1.6.0] - 2021-12-15
+### Added
+- Added optional <optgroup> label to DropdownItem
```
Comment (fleonard) [inline CHANGELOG.md:3]: you also need the link of the diff at the bottom of the file

Comment (joaocanaverde) [reply CHANGELOG.md:3]: @fleonard added

**Review by fleonard — APPROVED** [review]

**Pull request merged**